api/v3/Contribution.php

```diff
@@ -579,6 +579,8 @@ function _civicrm_api3_contribution_completetransaction_spec(&$params) {
 function civicrm_api3_contribution_repeattransaction($params) {
   $input = $ids = [];
   civicrm_api3_verify_one_mandatory($params, NULL, ['contribution_recur_id', 'original_contribution_id']);
+  // @todo Clean this up - we should pass through either original_contribution_id or contribution_recur_id
+  //   With a contribution_recur_id the template contribution will be calculated automatically.
```

Comment:
In fact we actually calculate the template contribution based on getTemplateContribution either way ....